tests/dbt/conftest.py

```diff
@@ -1,22 +1,19 @@
 from __future__ import annotations
 
 import typing as t
-from pathlib import Path
 
 import pytest
 
+from sqlmesh.core.context import Context
 from sqlmesh.dbt.context import DbtContext
+from sqlmesh.dbt.loader import DbtLoader
 from sqlmesh.dbt.project import Project
-from tests.conftest import delete_cache
 
 
 @pytest.fixture()
-def sushi_test_project() -> Project:
-    project_root = "tests/fixtures/dbt/sushi_test"
-    delete_cache(project_root)
-    project = Project.load(DbtContext(project_root=Path(project_root)))
-    for package_name, package in project.packages.items():
-        project.context.jinja_macros.add_macros(package.macro_infos, package=package_name)
```

Comment:
Why do we not need this any more?

Reply:
@crericha I believe this is from you (let me know if that isn't the case)

Reply:
This fixture would manually only do the project load and not all the other things the loader does, while the `sushi_test_dbt_context` fixture uses the lodaer. It made more sense to switch to that rather than extend this fixture with manual steps the loader already does.